### PR TITLE
feat(branding): Add icon based on theme docs

### PR DESCRIPTION
**Pull request by ChrisMcKee**
Relates to https://github.com/CoreWCF/CoreWCF/issues/1092

This is "possibly" correct; The gem bundle wouldn't install so I've not been able to properly check.

### CODE REVIEW

**Comment by ChrisMcKee** [comment]
@dotnet-policy-service agree

**Comment by mconnew** [comment]
I'll check this out locally and verify.

File: _config.yml [hunk 1]
```diff
@@ -30,7 +30,15 @@ github_username:  CoreWCF
 theme: minima
 plugins:
   - jekyll-feed
+  - jekyll-seo-tag
+
+header_pages:
   
+minima:
```
Comment (mconnew) [inline _config.yml:37]: I had trouble finding where you had got this from. I finally found it on the minima documenation in the projects GitHub repo. It's only for minima-3.0, which isn't available on rubygems.org, and also isn't backwards compatible so updating to it requires changing a bunch of other stuff. I did a bit of [digging in the jekyll-seo-tag docs](https://github.com/jekyll/jekyll-seo-tag/blob/v2.8.0/docs/usage.md) and there's a few things that can be done around this using minima-2.5.1. The following can be added for x/twitter, just needs to be placed after the existing github_username on line 27:
```
twitter_username: corewcf_
```
There's also a logo setting where you specify the path of the logo. I believe this is used for things like twitter summary cards.

Comment (ChrisMcKee) [reply _config.yml:37]: Was admittedly a wild stab in the dark. I use Hugo 😅

**Pull request closed without merging**